doc/en/components/charts/chart-features-animations.md

```diff
@@ -7,4 +7,61 @@ mentionedTypes: []
 
 # $PlatformShort$ Chart Animations
 
-<!-- category-chart-config-options.md -->
\ No newline at end of file
+Animations allows you to ease-in the series as it loads a new data source. The available animation differs depending on the type of series involved. For example, the column series animates by rising from the x-axis, a line series animates by drawing from the y-axis.
```

Comment:
change "drawing from the y-axis"
to "drawing from the origin of y-axis"

Reply:
pending